api/swagger.yaml

```diff
@@ -2870,6 +2870,14 @@ definitions:
             description: "Run an init inside the container that forwards signals and reaps processes. This field is omitted if empty, and the default (as configured on the daemon) is used."
             type: "boolean"
             x-nullable: true
+          PidsLimit:
```

Comment:
So, this is the only thing I'm not sure about; `PidsLimit` is directly in `TaskTemplate.ContainerSpec` (which is similar to how it's for container create), but other limits are in `TaskTemplate.Resources.Limits` / `TaskTemplate.Resources.Reservation`.

https://github.com/docker/swarmkit/pull/2415 was merged a long time ago, and implemented it like this on the SwarmKit side, so I wasn't sure if we should change it?